databricks_cli/pipelines/api.py

```diff
@@ -33,7 +33,7 @@
 
 BUFFER_SIZE = 1024 * 64
 base_pipelines_dir = 'dbfs:/pipelines/code'
-supported_lib_types = {'jar', 'whl'}
+supported_lib_types = {'jar', 'whl', 'maven'}
```

Comment:
add `egg` here as well?

Reply:
When we did python before we had a good reason for not supporting eggs. I don't remember exactly what that was, but we can do it in the future when it's needed.